Tasks/VsTest/vstest.ts

```diff
@@ -49,6 +49,7 @@ try {
     var useNewCollectorFlag = tl.getVariable('tia.useNewCollector');
     var isPrFlow = tl.getVariable('tia.isPrFlow');
     var vsTestVersionForTIA: number[] = null;
+	var ignoreVstestFailure: string = tl.getVariable("vstest.ignoretestfailures");
```

Comment:
This will only be consumed in mseng, and that too Phase 1 of reliability efforts. Sanjay & Siddhartha are already aware of this plan. Unless there is some recent development, that I am not aware of.